### ISSUE TITLE
Midterm Peer Review - jc3472

### ISSUE THREAD

**Issue by Ericland** [bz374/ORIE5147Project#10]
In this project, the team is trying to predict the income level based on the NLSY79 database. 

What I like about this project: 
1. The team clearly stated how they address the missing values. 
2. The team did some decent work on data visualization and justified their choices of features by showing that the chosen features have some correlations with the income level. 
3. The team clearly stated how they will split the dataset to training set and testing set to evaluate the generated models. 

Suggestions: 
1. Detailed description of the features is needed, including what each feature is, the type of each feature, the proposed encoding method of each feature, the amount of missing data of each feature, etc. 
2. Data visualization can be further improved. E.g., there are 15 features but only 3 features are visualized; axis title needs to be included; figure captions need to be included. 
3. Preliminary analysis can be further improved. Which part of the dataset has the highest impact on the prediction results? What models (linear regression, lasso, ridge, etc.) have been tried? What are their errors? 
4. Splitting data does not help reduce underfitting/overfitting directly. The team needs to propose more methods for addressing those issues (more complex model, feature pruning, etc.). Maybe the team mentioned their solutions somewhere but maybe it is better to use one paragraph to summarize their solutions.
